.github/actions/sync/shared-config.rb

```diff
@@ -97,18 +97,6 @@ def git(*args)
   mass-bottling-tracker-private
   ruby-macho
 ].freeze
-custom_dependabot_repos = %w[
-  .github
-].freeze
```

Comment:
I don't think you want to remove this as it overwrites the base template: #313

Reply:
@Bo98 on it, thanks.